sorc/IMSaggregate_mod.f90

```diff
@@ -79,7 +79,13 @@ subroutine calculate_scfIMS(idim, jdim, otype, yyyymmddhh, jdate, IMS_obs_path,
         real                :: oroFV3(idim,jdim,6) ! orography model grid
         character(len=250)  :: IMS_obs_file
         character(len=8)    :: date_from_file
-        integer             :: i,j,t
+        character(len=1)    :: tile_str,jstr1,istr1
+        character(len=2)    :: jstr2,istr2
+        character(len=3)    :: jstr3,istr3
+        character(len=4)    :: xind, yind, jstr4,istr4
+        character(len=9)    :: staids(idim,jdim,6) ! station ID string
```

Comment:
Suggest to use standard name `stid` to replace `staids`

Reply:
Corrected as suggested.